packages/dx-react-grid/src/plugins/table-header-row.tsx

```diff
@@ -45,9 +51,9 @@ export class TableHeaderRow extends React.PureComponent {
 
         <Template
           name="tableCell"
-          predicate={({ tableRow, tableColumn }) => isHeadingTableCell(tableRow, tableColumn)}
+          predicate={({ tableRow, tableColumn }: any) => isHeadingTableCell(tableRow, tableColumn)}
```

Comment:
Shouldn't it be `: Getters`?

Reply:
Yes, but then we need to change a signature for Template.predicate - now it accepts an object

Reply:
I'll make a separate PR